examples/python/extract-logs-write-to-cloud-logging/README.md

```diff
@@ -0,0 +1,66 @@
+## Overview
+
+A Python script that extracts Looker system/audit logs from [System Activity](https://docs.looker.com/admin-options/system-activity) and exports the Logs to Cloud Logging. This example tries to format the output logs like a [GCP Audit Log](https://cloud.google.com/logging/docs/audit/understanding-audit-logs) as best as possible. See [mapping](#gcp-audit-log-fields-to-looker-system-activity-mapping) for comparison between Looker System Activity Fields and GCP Audit Log Fields
+
+> **_NOTE:_**  The script extracts System Activity data from the last 10 minutes. You can then schedule this script to run every 10 minutes using a cron job or equivalent
+
+## Requirements
+- Looker Instance in which you have Admin or `see_system_activity` permission
+- Google Cloud Project with Cloud Logging API enabled
+- [pyenv](https://github.com/pyenv/pyenv#installation) installed
+
+## Deployment
+
+- Clone the repo and navigate to this directory
+  ```
+  git clone https://github.com/looker-open-source/sdk-codegen.git
+  cd sdk-codegen/examples/python/extract-logs-write-to-cloud-logging
+  ```
+
+- Setup Python Virtual environment 
+  ```
+  pyenv install 3.8.2
+  pyenv local 3.8.2
+  python -m venv .venv
+  ```
+
+- Install dependencies 
+  ```
+  pip install looker-sdk
+  pip install --upgrade google-cloud-logging
+  ```
+
+
+- Create API credentials and set environment variables
+  ```
+  export LOOKERSDK_BASE_URL="<Your API URL>"
+  export LOOKERSDK_CLIENT_ID="<Your Client ID>"
+  export LOOKERSDK_CLIENT_SECRET="<Your Client Secret>"
+  ```
+
+- Configure gcloud and [setup service account](https://cloud.google.com/logging/docs/reference/libraries#setting_up_authentication) to write Logs to Cloud Logging
+  ```
+  gcloud config set project <Project ID>
+  export GOOGLE_APPLICATION_CREDENTIALS="<Service Account Key Path>"
+  ```
+
+- Run `main.py`
+  ```
+  python main.py
+  ```
+
+
+## GCP Audit Log Fields to Looker System Activity Mapping
+
+| GCP Audit Log Field       | Looker System Actvity Field |
+| -----------               | -----------                 |
+| [logName](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#:~:text=Fields-,logName,-string) | `looker_system_activity_logs` |
```

Comment:
Do the GCP services that consume Audit Logs have requirements on the values in the Audit Log? 
This example would set precedent and probably be defacto standard for Audit logs coming from Looker.

Reply:
Yes GCP Audit logs have expected values.

However, some of these values do not apply or make sense for a Looker resource. For example, GCP resource always falls under a project/resource but a Looker resource doesn't. A project in Looker means something different. The mapping is on a best effort basis.

Technically the output of this script is not a GCP Audit log (when you filter for GCP Audit log, these logs won't appear). I'm only using the Audit log format in this example for similarity when querying these logs.

Reply:
Good to know. So this is only for querying standard GCP Audit Logs with Looker System activity Logs included. 
However, my main worry is compliance/auditing issues. Like i previously asked, is there any GCP services that  consume audit logs that may require specific values? (And have you consulted them on this)

Reply:
Yes, all GCP resources require certain values in audit logs(E.g Project, organization, IAM details). I already mentioned that some of these values don't apply to Looker because technically Looker is not a GCP resource (yet)

No, I did not consult with the Audit Logs team. This repo is only an example showing a way you can programmatically export System Activity logs  today, until the product team integrates Looker as a GCP service. It's my understanding that only the product team can get this type of compliance approval you're referring to.

System Activity logs is NOT a GCP Audit Log. And I'm sure the Audit team will echo that if I consult with them. This is not an official solution, but an example to demonstrate one of the workarounds you could do until the product team provides an official solution to this. 

I'm only mapping the SA logs to look like GCP Audit logs since most customers are familiar with the format and I mentioned in the readme that this is a best effort basis